### PR TITLE
remove clientgen

### LINKED ISSUE #35: Remove clientgen
For kcp-aware clientsets we've decided to go with the alternative approach of plumbing a kcp-aware round tripper into the existing kubernetes clients. As a result, the clientset generator here is no longer required and should be removed.

### PR DESCRIPTION
**Pull request by fabianvf**
closes #35

### CODE REVIEW

**Comment by fgiloux** [comment]
Hi @fabianvf the readme would also need to be updated.

**Comment by fabianvf** [comment]
@fgiloux right good catch

File: README.md [hunk 1]
```diff
@@ -19,29 +19,26 @@ It is possible to run the code generation for multiple components at once `code-
 **Note**: This is the relative path to the input directory where APIs live.
 
 2. `--output-dir` - The directory where output clients are to be generated. It defaults to the `clientset` folder under current working directory.
-    - `Clientset` wrappers would be generated inside `<outputDir>/<clientset-name>/clientset.go`.
-    - Individual typed client wrappers would be inside `<outputDir>/<clientset-name>/${GROUP}/${VERSION}/${group_version}.go`.
+    - Listers will be output to `<outputDur>/listers/${GROUP}/${VERSION}/${TYPE}.go`
```
Comment (ncdc) [inline README.md:22]:
```suggestion
    - Listers are output to `<outputDir>/listers/${GROUP}/${VERSION}/${TYPE}.go`
```

File: README.md [hunk 2]
```diff
@@ -19,29 +19,26 @@ It is possible to run the code generation for multiple components at once `code-
...
+    - Individual informers will be output to `<outputDur>/informers/externalversions/${GROUP}/${VERSION}/${TYPE}.go`
```
Comment (ncdc) [inline README.md:23]:
```suggestion
    - Individual informers are output to `<outputDir>/informers/externalversions/${GROUP}/${VERSION}/${TYPE}.go`
```

File: main.go [hunk 3]
```diff
@@ -47,19 +45,12 @@ func main() {
 	cmd := &cobra.Command{
 		Use:          "code-gen",
 		SilenceUsage: true,
-		Short:        "Generate cluster-aware kcp wrappers around clients, listers, and informers.",
-		Long:         "Generate cluster-aware kcp wrappers around clients, listers, and informers.",
-		Example: `Generate cluster-aware kcp clients from existing code scaffolded by k8.io/code-gen.
+		Short:        "Generate cluster-aware kcp wrappers around listers, and informers.",
```
Comment (ncdc) [inline main.go:48]:
```suggestion
		Short:        "Generate cluster-aware kcp wrappers around listers and informers.",
```

File: main.go [hunk 4]
```diff
@@ -47,19 +45,12 @@ func main() {
...
+		Long:         "Generate cluster-aware kcp wrappers around listers, and informers.",
```
Comment (ncdc) [inline main.go:49]:
```suggestion
		Long:         "Generate cluster-aware kcp wrappers around listers and informers.",
```

**Pull request merged**